### PR TITLE
Add profile

### PR DESCRIPTION
**Pull request by Varunmnx**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1709"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: public/data/VarunNarayanan.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      },
+      {
+         "name":"Follow me on Twitter",
+         "url":"https://twitter.com/Varun_Narayana1",
+         "icon":"twitter"
+      },
+      { "name": "Connect with me on LinkedIn", 
+        "url":"https://www.linkedin.com/in/varun-narayanan-74b653193", 
+        "icon": "linkedin" 
+      },
```
Comment (kumarsonsoff3) [inline public/data/VarunNarayanan.json:20]: Please remove this Trailing comma from here! 
```suggestion
      }
```

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by Varunmnx** [comment]
Done

**Review by kumarsonsoff3 — APPROVED** [review]
LGTM! 👍 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Pull request merged**